core/controller/project.js

```diff
@@ -34,10 +34,7 @@ route.get('/*', IsRole('BETA'), async (ctx, next) => {
 
 route.get('/init', async (ctx, next) => {
   const status = await ctx.project.getStatus()
-  .catch(err => ctx.throw({
-    message: 'Unable to get project status',
-    error: err
-  }, 500))
+  .catch(ctx.throw('Unable to get project status', 500))
```

Comment:
Do we really want to throw away error messages compleatly? We should at least be doing some kind of logging on them, no?